shell/src/main/java/alluxio/cli/fsadmin/FileSystemAdminShellUtils.java

```diff
@@ -37,6 +47,25 @@ public static int compareTierNames(String a, String b) {
     return aValue - bValue;
   }
 
+  /**
+   * Checks if the master client service is running.
```

Comment:
add that it will throw an exception if it fails to determine that the master client service is running

Reply:
Added, much clear, thanks